src/scenes/AccessionsRouter/edit/EndDryingReminderModal.tsx

```diff
@@ -56,45 +53,28 @@ export default function EndDryingReminderModal(props: EndDryingReminderModalProp
     }
   };
 
-  const onEnable = (id: string, value: boolean) => {
-    setEnable(value);
-    if (!value) {
-      setDate(undefined);
-      onChange('dryingEndDate', undefined);
-    }
-  };
-
-  const validate = (dateValue: string | undefined): boolean => {
+  const validateDate = (newValue?: string | number | undefined) => {
     setDateError('');
 
-    if (!dateValue) {
-      if (!enable) {
-        return true;
-      } else {
-        setDateError(strings.REQUIRED_FIELD);
-        return false;
-      }
-    }
-
-    const dateMs = new Date(dateValue).getTime();
-    const today = new Date(getTodaysDateFormatted()).getTime();
-
-    if (isNaN(dateMs)) {
+    // Invalid dates or blank entries will remove the reminder date
+    if (!newValue) {
       setDateError(strings.INVALID_DATE);
-      return false;
-    } else if (dateMs < today) {
+      return true;
+    } else if (!isInTheFuture(newValue)) {
```

Comment:
Using today's date shows an error of 'No past dates'. Maybe something like 'Must be in the future'?